components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -67,19 +68,41 @@
       name: {
         type: String,
         required: true
+      },
+      preSelected: {
```

Comment:
This seems to follow the opposite naming convention as the prop/data in MoreFacetsDropdown.vue. Is that accurate? If so, please align them.